### PR TITLE
CK-39 Create account creation page

### PR DESCRIPTION
**Pull request by JoshuaDueck**
* Created login page
* Added react-location for routing
* Added sign up page
* Removed top level package.json which was a result of accidental npm installing on the top directory.

### CODE REVIEW

**Review by Ahsen19 — APPROVED** [review]
LGTM
Awesome work 👏

**Review by Amikz — APPROVED** [review]
Looks good! Probably don't have to worry about it yet, but at some point we should make it so the user re-enters the password for sign-up (i.e. has to give their password twice)

File: frontend/src/components/pages/LogIn.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { useQuery } from "react-query";
+import { Link } from "@tanstack/react-location";
+
+const LogIn = () => {
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const username = e.target.username.value;
```
Comment (Amikz) [inline frontend/src/components/pages/LogIn.js:8]: should be email, not username?

File: frontend/src/components/pages/SignUp.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  return (
+    <>
+      <div>
+        <h1>Sign Up</h1>
+      </div>
+      <div>
+        <form onSubmit={(e) => handleSubmit(e)}>
+          <label>
+            Username:
```
Comment (Amikz) [inline frontend/src/components/pages/SignUp.js:25]: also should be email here

**Review by Tosvng — COMMENTED** [review]
Username should be changed to email. Apart from that everything looks great!

**Comment by JoshuaDueck** [comment]
Alright I made the changes y'all mentioned, plus a bit extra

File: frontend/src/components/pages/LogIn.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            Email:
+            <input type="email" name="email" />
+          </label>
+          <br />
+          <label>
+            Password:
+            <input
+              type="password"
+              name="password"
+              autoComplete="new-password"
```
Comment (Amikz) [inline frontend/src/components/pages/LogIn.js:36]: should be autoComplete="current-password"

File: frontend/src/components/pages/SignUp.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+            Email:
+            <input type="email" name="email" />
+          </label>
+          <br />
+          <label>
+            Password:
+            <input
+              type="password"
+              name="password"
+              autoComplete="current-password"
```
Comment (Amikz) [inline frontend/src/components/pages/SignUp.js:35]: should be new-password

**Review by Tosvng — APPROVED** [review]

File: frontend/src/components/pages/SignUp.js (hunk 4 above)
Comment (JoshuaDueck) [reply frontend/src/components/pages/SignUp.js:35]: Oh whoops, yep, you're right! I'm busy today but I'll fix that on Monday. Or you can pull the branch and make the changes if you need to work off of it today.

Comment (Amikz) [reply frontend/src/components/pages/SignUp.js]: I went ahead and make the changes

Comment (JoshuaDueck) [reply frontend/src/components/pages/SignUp.js]: Awesome, thanks!

**Pull request merged**